aten/src/THCUNN/generic/SpatialConvolutionMM.cu

```diff
@@ -157,8 +157,28 @@ void THNN_(SpatialConvolutionMM_updateOutput)(
   // Resize output
   THCTensor_(resize4d)(state, output, batchSize, nOutputPlane, outputHeight, outputWidth);
 
+  // we can bypass im2col in this case
+  bool direct_gemm = kW == 1 && kH == 1 && dW == 1 && dH == 1 && padW == 0 && padH == 0;
+  if (!direct_gemm) {
+    // Resize temporary columns
+    THCTensor_(resize2d)(state, columns, nInputPlane*kW*kH, outputHeight*outputWidth);
+  }
+
+  if (bias) {
+    // Define a buffer of ones, for bias accumulation
+    // Note: this buffer can be shared with other modules, it only ever gets increased,
+    // and always contains ones.
+    if (ones->dim() != 2 || ones->size(0)*ones->size(1) < outputHeight*outputWidth) {
+      // Resize plane and fill with ones...
+      THCTensor_(resize2d)(state, ones, outputHeight, outputWidth);
+      THCTensor_(fill)(state, ones, ScalarConvert<int, scalar_t>::to(1));
+    }
+  }
```

Comment:
I think you've forgot to remote line 178-190 by mistake. Let's remote these lines.

Reply:
My bad! Thanks for letting me know! I've just removed the lines. Please check again.